### PR TITLE
client/server: Add description to posts

### LINKED ISSUE #37: Add description field for new posts
Posts with long titles on gist work better than Drift, and rather than changing our styles I'd rather add a description field and enforce keeping titles on the shorter side.

### PR DESCRIPTION
**Pull request by jazcarate**
Closes #37

I'm very bad at frontend😅. I'd love some feedback on the description look

## List
image

## Creation
This is very bad 😂 
image

## Show
image

### CODE REVIEW

File: client/components/new-post/description/index.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const Description = ({ onChange, description }: props) => {
+	return (
+		<div className={styles.description}>
+			<Text h3>
+				Description
+			</Text>
+			<Textarea
+				value={description}
+				onChange={onChange}
+                width="100%"
```
Comment (MaxLeiter) [inline client/components/new-post/description/index.tsx:20]: can you run + commit `yarn lint` in client/?

**Review by MaxLeiter — CHANGES_REQUESTED** [review]

**Comment by MaxLeiter** [comment]
This is great! What are your thoughts on using an `<Input />` for the description to encourage keeping it on the shorter side? I envision descriptions being one sentence max, but maybe you have a different vision; I don't care a lot either way but my way may simplify the design.

**Comment by jazcarate** [comment]
I like the `<Input>` 😸 
image

**Comment by MaxLeiter** [comment]
Sorry @jazcarate, happy to merge this but the migration is `08_` now; I merged #25

File: client/components/new-post/description/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+type props = {
+	onChange: (e: ChangeEvent<HTMLInputElement>) => void
+	description?: string
+}
+
+const Description = ({ onChange, description }: props) => {
+	return (
+		<div className={styles.description}>
+			<Input
```
Comment (MaxLeiter) [inline client/components/new-post/description/index.tsx:14]: can we set the max length to 256? that's the same as Gist.

File: server/src/routes/posts.ts [hunk 3]
```diff
@@ -31,6 +31,7 @@ posts.post(
 	celebrate({
 		body: {
 			title: Joi.string().required(),
+			description: Joi.string().optional(),
```
Comment (MaxLeiter) [inline server/src/routes/posts.ts:34]:
```js       
Joi.string().optional()
    .min(0)
    .max(256)
```

Comment (MaxLeiter) [inline server/src/routes/posts.ts:34]:
```js       
Joi.string().optional()
    .min(0)
    .max(256)
```

**Review by MaxLeiter — CHANGES_REQUESTED** [review]

**Comment by MaxLeiter** [comment]
whoops, sorry for the delay on this @jazcarate. I think it's all set if you can fix the conflicts (again, sorry... I'm done with major changes for a little bit d:)

**Comment by jazcarate** [comment]
There! no more conflicts 😸

**Pull request merged**